e2_tts_pytorch/utils/compute_mel.py

```diff
@@ -0,0 +1,43 @@
+import torch
+from torch import nn
+import torchaudio
+class TorchMelSpectrogram(nn.Module):
+    def __init__(
+        self,
+        filter_length=1024,
+        hop_length=256,
+        win_length=1024,
+        n_mel_channels=80,
+        mel_fmin=0,
+        mel_fmax=8000,
+        sampling_rate=22050,
+        normalize=False,
+    ):
+        super().__init__()
+        self.filter_length = filter_length
```

Comment:
lines `16-23` can be omitted, as most of the arguments are forwarded to `MelSpectrogram`